src/modules/core/components/Slider/Slider.tsx

```diff
@@ -50,30 +52,48 @@ const Slider = ({
     }
   }, [sliderValue, value, setSliderValue]);
 
-  const gradientStopPercentage = useMemo(() => {
-    return limit ? Math.round((limit / max) * 100) : 0;
-  }, [limit, max]);
+  const limitValue = useMemo(() => {
+    return limit ? limit * 100 : 0;
```

Comment:
Limit might not always be `100`, I believe this should use the `max` value

Reply:
Limit is done in %, so it's passed like that: 0.36 and we need to multiply by 100 to get 36%. I don't think it should be max value here as it doesn't depend on it.

Reply:
Yes, but you don't know that it will be always a percent. _(Hence this is `core` component)_

So if sometime down the line we end up using this in some different scenario, we might use it with different numbers and not a percent based value.